src/main/java/it/gov/pagopa/pu/debtpositions/connector/organization/service/TaxonomyServiceImpl.java

```diff
@@ -1,15 +1,12 @@
 package it.gov.pagopa.pu.debtpositions.connector.organization.service;
 
 import it.gov.pagopa.pu.debtpositions.connector.organization.client.TaxonomySearchClient;
-import it.gov.pagopa.pu.organization.dto.generated.Taxonomy;
-import org.springframework.cache.annotation.CacheConfig;
-import org.springframework.cache.annotation.Cacheable;
+import it.gov.pagopa.pu.organization.dto.generated.PagedModelTaxonomy;
 import org.springframework.stereotype.Service;
 
-import java.util.Optional;
+import java.util.List;
 
 @Service
-@CacheConfig(cacheNames = it.gov.pagopa.pu.debtpositions.config.CacheConfig.Fields.taxonomy)
```

Comment:
we don't really want to cache it (we could use many method parameters to cache the result, avoiding to cache if empty page `#result.getEmbedded().getTaxonomies().size() == 0`  - to test executing the code, not with a junit!)?
in that case, let's remove it from properties, README and from Confluence page